registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/main/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessor.java

```diff
@@ -419,7 +421,7 @@ private void postAbisIdentification(InternalRegistrationStatusDto registrationSt
 			registrationStatusDto.setSubStatusCode(StatusUtil.BIO_DEDUPE_POTENTIAL_MATCH.getCode());
 			registrationStatusDto.setLatestTransactionStatusCode(RegistrationTransactionStatusCode.FAILED.toString());
 			moduleId = PlatformSuccessMessages.RPR_BIO_METRIC_POTENTIAL_MATCH.getCode();
-			packetInfoManager.saveManualAdjudicationData(matchedRegIds, registrationStatusDto.getRegistrationId(),
+			packetInfoManager.saveManualAdjudicationData(matchedRegIdsList, registrationStatusDto.getRegistrationId(),
```

Comment:
why is this changed done where matchedRegIdsList is not removed with its own reg ids ?

Reply:
here i changed the matchedRegIds type as Set (which having only unique regids, earlier getting repeated regids while testing correction packet) and other methods are expecting list so created list with same matchedRegIds  and forget to remove the own reg ids. 
Now i fixed this.